### PR TITLE
Claims UI

### PR DESCRIPTION
**Pull request by yanqiyqh**
Addressing issue #92, I updated the GUI to display our claims.

### CODE REVIEW

**Review by matthewyip1511 — APPROVED** [review]
LGTM! Proceeding to merge

File: src/main/java/seedu/address/model/client/Client.java [hunk 1]
```diff
@@ -17,6 +17,9 @@
  */
 public class Client {
 
+    public static final String NO_CLAIMS_MESSAGE = "No open claims";
+    public static final String OPEN_CLAIMS_MESSAGE = "%1$s open claims";
```
Comment (RezwanAhmed123) [inline src/main/java/seedu/address/model/client/Client.java:21]: nice use of wildcard strings!

File: src/main/java/seedu/address/model/client/Client.java [hunk 2]
```diff
@@ -82,6 +85,22 @@ public boolean isSameClient(Client otherClient) {
...
+     * If there are no open claims, it returns a predefined message indicating that there are no claims.
+     * Otherwise, it returns a formatted message displaying the count of open claims.
+     *
+     * @return A string message indicating the number of open claims. If there are no open claims,
+     *         it returns {@link #NO_CLAIMS_MESSAGE}. If there are open claims, it returns
+     *         a formatted string with the number of open claims using {@link #OPEN_CLAIMS_MESSAGE}.
+     */
+    public String getClaimsString() {
+        int numberOfOpenClaims = this.insurancePlansManager.getNumberOfOpenClaims();
+        return (numberOfOpenClaims == 0) ? NO_CLAIMS_MESSAGE : String.format(OPEN_CLAIMS_MESSAGE, numberOfOpenClaims);
```
Comment (RezwanAhmed123) [inline src/main/java/seedu/address/model/client/Client.java:101]: this notation makes the code more readable 👍

**Review by RezwanAhmed123 — COMMENTED** [review]
LGTM!

**Pull request merged**